packages/main-api/src/controllers/v1/mod.rs

```diff
@@ -85,5 +87,11 @@ pub fn route(pool: sqlx::Pool<sqlx::Postgres>) -> Result<by_axum::axum::Router>
         .nest(
             "/subscriptions",
             subscriptions::SubscriptionController::new(pool.clone()).route(),
+        ).nest(
+            "/followers",
+            followers::FollowerController::new(pool.clone()).route()?,
+        ).nest(
+            "/following",
+            following::FollowingController::new(pool.clone()).route()?,
```

Comment:
you should implement actions in `/v1/followers` which is described in `base` of `follower.rs`.
But I recommend you to add actions(`follow` and `unfollow`) in `/v1/users` of `/v1/my-network`